tests/zero_code_change/tensorflow2_integration_tests.py

```diff
@@ -88,6 +191,11 @@ def test_keras_v2(script_mode: bool = False, eager_mode: bool = True):
     script_mode = args.script_mode
 
     # eager mode
-    test_keras_v2(script_mode=script_mode)
+    test_keras_v2_default(script_mode)
+    test_keras_v2_multi_collections(script_mode)
+    test_keras_v2_save_all(script_mode)
+
     # non-eager mode
-    test_keras_v2(script_mode=script_mode, eager_mode=False)
+    test_keras_v2_default(script_mode, eager_mode=False)
+    test_keras_v2_multi_collections(script_mode, eager_mode=False)
+    test_keras_v2_save_all(script_mode, eager_mode=False)
```

Comment:
Might be able to trim down the test logic by just testing the save_all case. There are unit tests to ensure that collections work properly, this file should really just focus on the ZCC aspects.

Reply:
the multi collections case was the one that failed on the previous PR. so explicitly added a test for that case. might be redundant but thought to have it in there for now. what do you think?